js/components/ps-number-input.js

```diff
@@ -0,0 +1,101 @@
+/**
+ * Permit to create an input number element as PrestaShop wants it
+ *
+ * @param {DOM Element Object} element
+ * @method validate
+ * @method updateValue
+ * @returns {Object}
+ */
+const PSNumberInput = function PSNumberInput(element) {
+    this.min = Number(element.dataset.min);
+    this.max = Number(element.dataset.max);
+    this.labelMin = element.dataset.labelMin;
+    this.labelMax = element.dataset.labelMax;
+    this.labelNan = element.dataset.labelNan;
+    this.incrementButton = element.querySelector('.ps-number-increment');
+    this.decrementButton = element.querySelector('.ps-number-decrement');
+    this.input = element.querySelector('input');
+    this.invalidElement = element.querySelector('.invalid-feedback');
+
+    this.initValue = () => {
+        if(this.input.value.replace('.', '') !== this.input.value || this.input.value.replace(',', '') !== this.input.value) {
```

Comment:
You should add eslint :thinking:

Reply:
It will be a part of another PR, not here